enzyme/test/ActivityAnalysis/integration.ll

```diff
@@ -0,0 +1,93 @@
+; RUN: if [ %llvmver -gt 16 ]; then %opt < %s %newLoadEnzyme -passes="print-activity-analysis" -activity-analysis-func=f.preprocess-S | FileCheck %s
```

Comment:
Hello @wsmoses, could you let me know how I can run a test file only for a llvm version greater than a certain number and still pass the test when this number requirement is not met? Same issue with https://github.com/EnzymeAD/Enzyme/pull/1634